### ISSUE TITLE
Add rules for ARIA tables / grid

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#2457]
Axe-core has no rules that test ARIA grid / tables like this:

```html
<div role="grid">
	<div role="row">
		<div role="columnheader">Room</div>
		<div role="columnheader">Occupant</div>
	</div>
	<div role="row">
		<div role="gridcell">1A</div>
	</div>
	<div role="row">
		<div role="gridcell">2A</div>
	</div>
</div>
```

**Comment by straker** [comment]
Could you expand on what the rule would check? Our aira-required-children/parent rules already check the structure is correct.

**Comment by WilcoFiers** [comment]
Specifically, run the ARIA equivalent of th-has-data-cell. We're currently testing this for native tables, but not for ARIA.

Low priority though, this the native table rules only reports incompletes.
